wandb/sdk/wandb_run.py

```diff
@@ -1998,33 +2013,38 @@ def _create_job(self) -> None:
         installed_packages_list = sorted(
             [f"{d.key}=={d.version}" for d in iter(pkg_resources.working_set)]
         )
-        if has_repo:
-            artifact = self._create_repo_job(
-                input_types, output_types, installed_packages_list
-            )
-        elif self._code_artifact:
-            artifact = self._create_artifact_job(
+
+        for job_creation_function in [
```

Comment:
do we want to track in case all failed (or we just ignore it silently)?

Reply:
I can log it